manage.js

```diff
@@ -0,0 +1,87 @@
+
+var express = require('express');
+var app = express();
+var Microsoft = require('./luis-add');
+var Re =require('./luis-read');
+var De =require('./luis-delete');
+var Up2 =require('./luis-update');
+var Google = require('./dialogflow-add');
+var Ea =require('./dialogflow-read');
+var Dd =require('./dialogflow-delete');
+var Up1 =require('./dialogflow-update');
+Mi = new Microsoft();
+re =new Re();
+de =new De();
+up2 =new Up2();
+Go = new Google();
+di =new Ea();
+dd =new Dd();
+up =new Up1();
+var bodyParser = require('body-parser');
+var urlencodedParser = bodyParser.urlencoded({ extended: false });
+app.use(express.static('public'));
```

Comment:
I would suggest you to have `bodyParser.json()` in the express server. Usually web apps communicate with each other using `application/json` format. So add one line here:

```javascript
app.use(bodyParser.json());
```